README.md

```diff
@@ -15,24 +15,24 @@ rounds (eg. > 30 rounds).
     scala>  import com.github.t3hnar.bcrypt._
     import com.github.t3hnar.bcrypt._
 
-    scala>  "password".bcrypt
-    res1: String = $2a$10$iXIfki6AefgcUsPqR.niQ.FvIK8vdcfup09YmUxmzS/sQeuI3QOFG
+    scala>  "password".boundedBcryptSafe
```

Comment:
can we call it `bcryptBounded` ? so prefix remained the same comparing to old method?

Reply:
Of course